// ethereum-multicall/dist/esm/enums/networks.js
export var Networks;
(function (Networks) {
    Networks[Networks["mainnet"] = 1] = "mainnet";
    Networks[Networks["ropsten"] = 3] = "ropsten";
    Networks[Networks["rinkeby"] = 4] = "rinkeby";
    Networks[Networks["goerli"] = 5] = "goerli";
    Networks[Networks["optimism"] = 10] = "optimism";
    Networks[Networks["kovan"] = 42] = "kovan";
    Networks[Networks["bsc"] = 56] = "bsc";
    Networks[Networks["kovanOptimism"] = 69] = "kovanOptimism";
    Networks[Networks["bsc_testnet"] = 97] = "bsc_testnet";
    Networks[Networks["xdai"] = 100] = "xdai";
    Networks[Networks["matic"] = 137] = "matic";
    Networks[Networks["mumbai"] = 80001] = "mumbai";
    Networks[Networks["avalancheFuji"] = 43113] = "avalancheFuji";
    Networks[Networks["avalancheMainnet"] = 43114] = "avalancheMainnet";
    Networks[Networks["etherlite"] = 111] = "etherlite";
    Networks[Networks["arbitrum"] = 42161] = "arbitrum";
    Networks[Networks["fantom"] = 250] = "fantom";
    Networks[Networks["cronos"] = 25] = "cronos";
    Networks[Networks["harmony"] = 1666600000] = "harmony";
})(Networks || (Networks = {}));
